web/src/components/schedule/schedule.js

```diff
@@ -1,5 +1,10 @@
-function Schedule() {
-  return <div>Schedule!</div>;
-}
+import { Box } from "@mui/material";
+import ClassesOverview from "./subcomponents/classes_overview/classes_overview";
+import indexStyles from "index.styles";
```

Comment:
Instead of putting this here, why don't we just put it in index.js? Like put a div around the router and apply the style to that?